cfdm/read_write/netcdf/netcdfwrite.py

```diff
@@ -18,6 +18,18 @@
 class NetCDFWrite(IOWrite):
     """A container for writing Fields to a netCDF dataset."""
 
+    def __new__(cls, *args, **kwargs):
+        """Store the NetCDFRead class.
+
+        .. note:: If a child class requires a different NetCDFRead class
+        than the one defined here, then it must be redefined in the
+        child class.
```

Comment:
Just thinking this comment is probably expressing something quite obvious? I guess you want to stress this, in which case it might be worth including, otherwise I'd be tempted to drop it?

Reply:
Fixed in https://github.com/NCAS-CMS/cfdm/pull/196/commits/8d8d6949d76351ac36090a86971a82eacb363ac2